README.md

```diff
@@ -56,6 +56,27 @@ $ helm install devfile-registry ./deploy/chart/devfile-registry \
 	--set devfileIndex.tag=latest
 ```
 
+You can deploy a devfile registry with a registry viewer image (uses `quay.io/devfile/registry-viewer:next` by default) by running the following:
```

Comment:
Okay, so this is my bad 🤦 . I know I said to exclude "custom" from "custom registry viewer image", but after looking at the full markdown file with the other examples provided, I think it makes sense to say "custom registry viewer image"